packages/pluggableWidgets/timeline-web/package.json

```diff
@@ -3,7 +3,7 @@
   "widgetName": "Timeline",
   "version": "2.0.0",
   "description": "Shows timeline",
-  "copyright": "Mendix BV",
+  "copyright": "Copyright © 2005-2020 Mendix Technology B.V. All rights reserved.",
```

Comment:
We should edit this for all widgets

Reply:
I agree

Reply:
This would need to be changed to 2021 first, for all widgets, before we merge this PR

Reply:
Probably better to check if the string is malformed, then write something up-to-date